_pages/policies/employee-resources-policies/hiring.md

```diff
@@ -19,46 +19,40 @@ For more information on hiring via the Competitive Service (Career) Merit Promot
 
 The following explains the process for initiating TTS internal competitions (e.g. directors, supervisors) and developmental opportunities (e.g. details, leads).
 
-For detailed information specific to initiating an internal or external detail, please review the [details page]({{site.baseurl}}/assignee-detail) of the handbook.
+For detailed information specific to initiating an internal or external detail, please review the [details page]({{site.baseurl}}/assignee-detail) of the handbook.<br>
```

Comment:
I don't think the BR should be necessary, should it? You should get a paragraph break just from the line breaks, right?

Reply:
probably? not sure why i did that...